lib/logstash/filters/zeromq.rb

```diff
@@ -171,7 +171,7 @@ def send_recv(message)
 
   public
   def filter(event)
-    return unless filter?(event)
+    
```

Comment:
this might be a limitation of the mass update script, but would be nicer to not add an empty line